gamestonk_terminal/stocks/options/op_helpers.py

```diff
@@ -124,3 +128,149 @@ def rn_payoff(x: str, df: pd.DataFrame, put: bool, delta: int, rf: float) -> flo
     "openInterest": {"format": "", "label": "Open Interest"},
     "impliedVolatility": {"format": "{x:.2f}", "label": "Implied Volatility"},
 }
+
+
+class Option:
+    def __init__(
+        self,
+        s: float,
+        k: float,
+        rf: float,
+        div_cont: float,
+        expiry: float,
+        vol: float,
```

Comment:
Should be able to create a method that finds IV from the strike price

Reply:
This is handled in the view.